ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -20,19 +20,18 @@ OnboardingStackView {
     required property int restoreKeysExportState
     required property int addKeyPairState
     required property int syncState
-    required property var generateMnemonic
     required property int remainingPinAttempts
     required property int remainingPukAttempts
 
-    required property bool isBiometricsLogin // FIXME should come from the loginAccountsModel for each profile separately?
-
     required property bool biometricsAvailable
     required property bool displayKeycardPromoBanner
     required property bool networkChecksEnabled
 
     property int keycardPinInfoPageDelay: 2000
 
     // functions
+    required property var generateMnemonic
+    required property var isBiometricsLogin
```

Comment:
```suggestion
    required property bool isBiometricsLogin
```

Reply:
it's actually function there

Reply:
We should add better descriptions to that and all other functions in the api...